src/Data/HashTable/ST/Basic.hs

```diff
@@ -608,6 +665,10 @@ deletedMarker = 1
 
 
 ------------------------------------------------------------------------------
+trueInt :: Int -> Bool
```

Comment:
Ok. I asked only because `GHC.Prim` documentation tells to "Use `GHC.Exts` from the base package instead of importing this module directly." and I wasn't sure whether this situation grants an exception (especially since `hashtables` package doesn't have `ghc-prim` as a dependency yet). Thanks for clearing this up.